### PR TITLE
feat: Manage .env files using bucket

### PR DESCRIPTION
**Pull request by hugotiburtino**

### CODE REVIEW

**Comment by elbotho** [comment]
I love how little code changes that needed 👍 
Did you check deployment already as well?

**Comment by hugotiburtino** [comment]
> Did you check deployment already as well?

Thanks for calling my attention to that. Now it was succesfully tested.

File: README.md [hunk 1]
```diff
@@ -42,3 +43,28 @@ express.
...
+
+To use CLI to update any environment variable of the development, staging and
+production environments, follow these steps:
+
+1. Ask the admin to include you into the IONOS contract and update to policy of
+   the corresponding bucket. Alternatively, you can use the credentials of the
+   dev or admin user.
+2. Install a s3 client CLI (we recommend `s3cmd`) and configure it accordingly.
+   For access and secret keys go to
+   https://dcd.ionos.com/latest/#/key-management. Some other info:
```
Comment (LarsTheGlidingSquirrel) [inline README.md:76]: I cannot see the bucket when logging in using the `dev@serlo.org` account.

File: README.md [hunk 2]
```diff
@@ -42,3 +43,28 @@ express.
 
 On a successful LTI launch the server returns a signed `accessToken` jwt that
 the client can then later use to authenticate saving content.
+
+# Management of Env Vars of the Deployments
```
Comment (LarsTheGlidingSquirrel) [inline README.md]: Should this be done on the server or locally? Maybe add a comment about this here.

**Review by LarsTheGlidingSquirrel — COMMENTED** [review]
Thank you! I tried it out but got stuck. See comments.

File: README.md (hunk 1 above)
Comment (elbotho) [reply README.md:76]: try switching to "contract-owned buckets", then it should show.

File: README.md (hunk 2 above)
Comment (hugotiburtino) [reply README.md]: I've updated the readme. Take a look if it is clearer now please.

File: README.md (hunk 1 above)
Comment (LarsTheGlidingSquirrel) [reply README.md:76]: Ah nice! I did not see this small dropdown. Thank you!

File: README.md (hunk 2 above)
Comment (LarsTheGlidingSquirrel) [reply README.md]: Now it is clear! Thank you for updating it! :green_heart:

**Review by LarsTheGlidingSquirrel — APPROVED** [review]
I did some testing and was able to update the .env file both in Ionos UI and CLI. Thank you for finding a better way to store the env values.

**Pull request merged**